src/core/web-server.cc

```diff
@@ -41,6 +41,7 @@ class VramExecutor : public PCSX::WebExecutor {
         static constexpr uint32_t texSize = 1024 * 512 * sizeof(uint16_t);
         uint16_t* pixels = (uint16_t*)malloc(texSize);
         int oldTexture;
+        glFlush();
```

Comment:
Honestly, we really just need to get this the fuck out of the web server. Doing "OpenGL" here is heresy; I really just took a shortcut that I shouldn't have. We rather should have a "get me a Slice that represents the VRAM" query method in the GPU object or something.

Not sure if you want to do it as part of your cleanup tho.